package.json

```diff
@@ -1040,6 +1040,7 @@
         "webpack-cli": "^3.1.2"
     },
     "dependencies": {
+        "@azure/cosmos": "^3.6.3",
         "@azure/arm-cosmosdb": "^9.1.0",
```

Comment:
Can you remove "documentdb" and "@types/documentdb" now? Also we probably don't need "@azure/cosmos" listed in devDependencies now that it's in "dependencies"

Reply:
Thank you, I missed to remove them. Will do!